test/ds/test_dynamic_shape_models.py

```diff
@@ -42,6 +43,27 @@ def forward(self, x):
     return output
 
 
+class CNN(torch.nn.Module):
```

Comment:
I am glad to see DS passes this test. Did you verify recompilation is flat after the first pass?

cc @JackCaoG